src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const list = document.querySelectorAll('.population');
+const total = [...list]
+  .map(element => Number(element.innerHTML.replace(/,/g, '')))
+  .reduce((acc, population) => acc + population, 0);
+const average = total / [...list].length;
+
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+totalPopulation.innerHTML = total.toLocaleString();
+averagePopulation.innerHTML = average.toLocaleString();
```

Comment:
You should return the number in the format 452,543,964 instead of 452 543 964.
Try to change locale to 'en-US'